ExcelToDBTransferer/PersonDTO.cs

```diff
@@ -9,14 +9,16 @@ public class PersonDTO
     public string LastName { get; }
     public string Initials { get;}
     public bool IsActive { get; }
+    public string LoginName { get; }
 
-    public PersonDTO(string firstName, string lastName, string initials, bool isActive)
+    public PersonDTO(string firstName, string lastName, string initials, bool isActive, string loginName)
     {
```

Comment:
Shouldn't this be nullable as we wanted inactive users to not have a login name?